### ISSUE TITLE
 Plot xxx.plot has a farmer public key that is not in the farmer's pk list.

### ISSUE THREAD

**Issue by codeyee** [madMAx43v3r/chia-plotter#435]
Unable to detect my plot file using "Chia Start Harvester",The error message was "Plot XXX. Plot has a farmer public key that is not in the farmer's PK list.", but I used" Chia plots to check" it was normal 

![image](https://user-images.githubusercontent.com/43313765/121877591-5b190e80-cd3d-11eb-8c23-37050bfbec7a.png)

![image](https://user-images.githubusercontent.com/43313765/121877603-5e13ff00-cd3d-11eb-911f-65dda21e6a7f.png)

thx

**Comment by madMAx43v3r** [comment]
then you set wrong keys when plotting, make sure farmer and public key are not flipped

**Comment by codeyee** [comment]
> then you set wrong keys when plotting, make sure farmer and public key are not flipped

Yes, I sure farmer and public key are not flipped, these are my running parameters

```shell
chia_plot -n 1 -r 4 -t /mnt/ssd/ -d /mnt/ssd/ -p a2f9...ce0a -f 817e...5969
```

**Comment by madMAx43v3r** [comment]
`chia plots check` will just show whatever you give `chia_plot`, you have to compare to `chia keys show`.
